SortColors.java

```diff
@@ -0,0 +1,18 @@
+class SortColors{
+    public void sortColors(int[] nums) {
+        int count0 = 0, count1 = 0, count2 = 0;
```

Comment:
Please also try the **Three Pointers approach, one pass** to solve this problem.